### ISSUE TITLE
Guru Meditation Error of type

### ISSUE THREAD

**Issue by albersag** [Jeija/esp32free80211#5]
Hi

Im getting this log and error. Should it be caused by new wifi library?

Can you give me some times how to downgrade to your working one?

Regards

ets Jun  8 2016 00:22:57

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
flash read err, 1000
Falling back to built-in command interpreter.
OK
>ets Jun  8 2016 00:22:57

rst:0x10 (RTCWDT_RTC_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0x00
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0008,len:8
load:0x3fff0010,len:3740
ho 0 tail 12 room 4
load:0x40078000,len:12116
load:0x40080000,len:252
entry 0x40080034
check b=0x3ffae2ac size=7496 ok
check b=0x3ffafff4 size=0 ok
check b=0x3ffb733c size=167096 ok
check b=0x3ffdfff4 size=0 ok
check b=0x3ffe044c size=15272 ok
check b=0x3ffe3ff4 size=0 ok
check b=0x3ffe435c size=113816 ok
check b=0x3ffffff4 size=0 ok
check b=0x40092d30 size=53956 ok
D (739) nvs: nvs_flash_init_custom start=9 count=6[0m
D (742) nvs: nvs_open misc 1[0m
D (742) nvs: nvs_get_str_or_blob log[0m
I (743) wifi: wifi firmware version: 6c86a1c
I (744) wifi: config NVS flash: enabled
I (748) wifi: config nano formating: disabled
D (752) nvs: nvs_open nvs.net80211 1[0m
D (756) nvs: nvs_get opmode 1[0m
D (759) nvs: nvs_get country 1[0m
D (762) nvs: nvs_get_str_or_blob sta.ssid[0m
D (766) nvs: nvs_get_str_or_blob sta.mac[0m
D (770) nvs: nvs_get sta.authmode 1[0m
D (773) nvs: nvs_get_str_or_blob sta.pswd[0m
D (777) nvs: nvs_get_str_or_blob sta.pmk[0m
D (781) nvs: nvs_get sta.chan 1[0m
D (785) nvs: nvs_get auto.conn 1[0m
D (788) nvs: nvs_get bssid.set 1[0m
D (791) nvs: nvs_get_str_or_blob sta.bssid[0m
D (795) nvs: nvs_get sta.phym 1[0m
D (799) nvs: nvs_get sta.phybw 1[0m
D (802) nvs: nvs_get_str_or_blob sta.apsw[0m
D (806) nvs: nvs_get_str_or_blob sta.apinfo[0m
D (811) nvs: nvs_get_str_or_blob ap.ssid[0m
D (814) nvs: nvs_get_str_or_blob ap.mac[0m
D (818) nvs: nvs_get_str_or_blob ap.passwd[0m
D (822) nvs: nvs_get_str_or_blob ap.pmk[0m
D (826) nvs: nvs_get ap.chan 1[0m
D (829) nvs: nvs_get ap.authmode 1[0m
D (833) nvs: nvs_get ap.hidden 1[0m
D (836) nvs: nvs_get ap.max.conn 1[0m
D (840) nvs: nvs_get bcn.interval 2[0m
D (843) nvs: nvs_get ap.phym 1[0m
D (846) nvs: nvs_get ap.phybw 1[0m
D (849) nvs: nvs_get ap.sndchan 1[0m
D (853) nvs: nvs_set_blob sta.mac 6[0m
D (857) nvs: nvs_set_blob ap.mac 6[0m
I (860) wifi: Init dynamic tx buffer num: 32
I (864) wifi: Init dynamic rx buffer num: 64
I (868) wifi: wifi driver task: 3ffbb45c, prio:23, stack:4096
I (873) wifi: Init static rx buffer num: 10
I (877) wifi: Init dynamic rx buffer num: 0
I (881) wifi: Init rx ampdu len mblock:7
I (885) wifi: Init lldesc rx ampdu entry mblock:4
I (889) wifi: wifi power manager task: 0x3ffc09c4 prio: 21 stack: 2560
I (1552) wifi: wifi timer task: 3ffc1aa4, prio:22, stack:3584
D (1553) nvs: nvs_open phy 0[0m
D (1553) nvs: nvs_get cal_version 4[0m
D (1553) nvs: nvs_get_str_or_blob cal_mac[0m
D (1558) nvs: nvs_get_str_or_blob cal_data[0m
D (1562) nvs: nvs_close 3[0m
I (1578) wifi: mode : softAP (24:0a:c4:06:ce:81)
D (1580) tcpip_adapter: check: remote, if=1 fn=0x4010c4a4
[0m
I (1580) wifi: sleep disable
26 8 01 Never gonna give you up

Guru Meditation Error of type LoadProhibited occurred on core  0. Exception was unhandled.
Register dump:
PC      : 0x400e6645  PS      : 0x00060c30  A0      : 0x800fad9e  A1      : 0x3ffc3160  
A2      : 0x3ffb5824  A3      : 0x3ffaedd4  A4      : 0x00000000  A5      : 0x00000060  
A6      : 0x3ffc1c2c  A7      : 0x3ffb5f50  A8      : 0x00000000  A9      : 0x00053000  
A10     : 0x3ffaee14  A11     : 0x00000001  A12     : 0x00000053  A13     : 0x3ffaee6c  
A14     : 0x00000000  A15     : 0x01040501  SAR     : 0x0000001e  EXCCAUSE: 0x0000001c  
EXCVADDR: 0x00000000  LBEG    : 0x4000c2e0  LEND    : 0x4000c2f6  LCOUNT  : 0x00000000  

Backtrace: 0x400e6645:0x3ffc3160 0x400fad9e:0x3ffc3190 0x400dbcca:0x3ffc31b0

Entering gdb stub now.
$T0b#e6

**Comment by Jeija** [comment]
Yes, this happens because the esp32-wifi-lib version you are using is newer than the one esp32free80211 is compatible with. You have to downgrade esp-idf and esp32-wifi-lib using the appropriate `git reset` commands. For instance, in order to downgrade esp32-wifi-lib you can use `git reset --hard ffe5a4c14fe9c933c776fadc62fa9d409929e6f9` (this commit is mentioned in the README.md of this probject) in the esp32-wifi-lib directory. Make sure to also downgrade esp-idf to a version that was released when that commit was made, e.g. `git reset --hard af843a951705c27e854b39057bc3cedbf892b4d9` works for me.
